src/scene/forward-renderer.js

```diff
@@ -839,6 +844,135 @@ Object.assign(ForwardRenderer.prototype, {
         }
     },
 
+    // placeholder LUT textures for area light
+    _createAreaLightPlaceholderLuts: function () {
+        var placeholderLutTex =  new Texture(this.device, {
+            width: 2,
+            height: 2,
+            format: PIXELFORMAT_R8_G8_B8_A8
+        });
+        placeholderLutTex.name = 'placeholder';
+
+        var pixels = placeholderLutTex.lock();
+        pixels.fill(0);
```

Comment:
Not available in IE11: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/fill#browser_compatibility

Reply:
Will Babel add polyfill for this automatically?

Reply:
No. This:

```
class Mat4 {
    constructor() {
        var data = new Float32Array(16);
        data.fill(0);
        // Create an identity matrix. Note that a new Float32Array has all elements set
        // to zero by default, so we only need to set the relevant elements to one.
        data[0] = data[5] = data[10] = data[15] = 1;
        this.data = data;
    }
```

Becomes:

```
	var Mat4 = function () {
		function Mat4() {
			var data = new Float32Array(16);
			data.fill(0);
			data[0] = data[5] = data[10] = data[15] = 1;
			this.data = data;
		}
```

Reply:
I replaced the `fill()` call with a nested for loop (copied from the material handler) - which should work with ie11.